pkg/operator/ceph/csi/template/nfs/csi-nfsplugin.yaml

```diff
@@ -49,37 +50,76 @@ spec:
               mountPath: /registration
         - name: csi-nfsplugin
           securityContext:
-            allowPrivilegeEscalation: true
-            capabilities:
-              add:
-              - SYS_ADMIN
             privileged: true
+            capabilities:
+              add: ["SYS_ADMIN"]
+            allowPrivilegeEscalation: true
+          image: {{ .CSIPluginImage }}
           args:
-          - "--v={{ .LogLevel }}"
-          - "--drivername={{ .DriverNamePrefix }}nfs.csi.ceph.com"
-          - --nodeid=$(NODE_ID)
-          - --endpoint=$(CSI_ENDPOINT)
+            - "--nodeid=$(NODE_ID)"
+            - "--type=nfs"
+            - "--endpoint=$(CSI_ENDPOINT)"
+            - "--v={{ .LogLevel }}"
+            - "--nodeserver=true"
+            - "--drivername={{ .DriverNamePrefix }}nfs.csi.ceph.com"
+            - "--pidlimit=-1"
           env:
-          - name: NODE_ID
-            valueFrom:
-              fieldRef:
-                apiVersion: v1
-                fieldPath: spec.nodeName
-          - name: CSI_ENDPOINT
-            value: unix:///csi/csi.sock
-          image: {{ .NFSPluginImage }}
-          imagePullPolicy: IfNotPresent
+            - name: POD_IP
```

Comment:
Do we need POD_IP and POD_NAMESPACE for nfs?

Reply:
removed